### PR TITLE
this commit adds new setter module allowing for declarative approach …

### PR DESCRIPTION
**Pull request by johndagostino**
…to defining custom setters for properties

### CODE REVIEW

File: test/twin/setter_test.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require "test_helper"
+
+class SetterTest < Minitest::Spec
+  Composer = Struct.new(:name)
+
+  class Twin < Disposable::Twin
+    feature Setter
```
Comment (johndagostino) [inline test/twin/setter_test.rb]: @apotonick not sure if you want this feature by default, or optional

**Comment by johndagostino** [comment]
@apotonick build fails on proc syntax on ruby 1.9.3 in test

**Comment by JuanitoFatas** [comment]
> @apotonick build fails on proc syntax on ruby 1.9.3 in test

Remove space between `->` and `(` should work.

**Comment by johndagostino** [comment]
@JuanitoFatas great thanks!. @apotonick build fixed.

**Comment by apotonick** [comment]
Looks fantastic (both you and the PR)! :heart:

**Comment by apotonick** [comment]
@JuanitoFatas Cool, I totally didn't know about the `->()` issue, thanks!!! :beer: 

BTW, is Rails Pacific happ'ning this year?

**Comment by JuanitoFatas** [comment]
> @JuanitoFatas Cool, I totally didn't know about the ->() issue, thanks!!!

:blush: 

> BTW, is Rails Pacific happ'ning this year?

I hope! because it is the only way for me to see you :wink:

/cc @xdite

**Comment by apotonick** [comment]
Why did we need this, again?

**Comment by johndagostino** [comment]
@apotonick i don't remember.

**Comment by johndagostino** [comment]
@apotonick calculated defaults in reform?

**Pull request closed without merging**